### PR TITLE
Ensure that pokes are always send only via DFRN

### PR DESCRIPTION
**Pull request by annando**
This ensures that pokes are sent via DFRN, even if the contact is a newer Friendica contact that primarily uses AP. 

In a future step we should have a look at the different deliver commands. I guess we can reduce many of them to the "POST" constant.

### CODE REVIEW

File: src/Protocol/Diaspora.php [hunk 1]
```diff
@@ -2148,9 +2149,9 @@ private static function receiveParticipation(array $importer, $data)
 				continue;
 			}
 			if ($comment['verb'] == ACTIVITY_POST) {
-				$cmd = $comment['self'] ? 'comment-new' : 'comment-import';
+				$cmd = $comment['self'] ? Delivery::COMMENT : 'comment-import';
 			} else {
-				$cmd = $comment['self'] ? 'like' : 'comment-import';
```
Comment (nupplaphil) [inline src/Protocol/Diaspora.php]: just to be sure .. so `like` is now intenationally replaced with `activtiy-new` .. I don't understand the content well enough, but this is something I noticed

Comment (annando) [reply src/Protocol/Diaspora.php]: Yeah. I just unified this. But in fact the content here doesn't matter much. It is important for some special stuff. But except from this we could put everything there.

File: src/Worker/APDelivery.php [hunk 2]
```diff
@@ -37,6 +37,7 @@ public static function execute($cmd, $target_id, $inbox, $uid)
 		} elseif ($cmd == Delivery::SUGGESTION) {
 			$success = ActivityPub\Transmitter::sendContactSuggestion($uid, $inbox, $target_id);
 		} elseif ($cmd == Delivery::RELOCATION) {
+		} elseif ($cmd == Delivery::POKE) {
```
Comment (MrPetovan) [inline src/Worker/APDelivery.php]: Please move the condition with empty blocks to the final `else` like this:
```php
} elseif (!in_array($cmd, [Delivery::RELOCATION, Delivery::POKE]) {
```

Comment (annando) [reply src/Worker/APDelivery.php]: This is some kind of reminder to myself to fill these empty lines. And especially the relocation is planned to be realized for AP.

**Pull request merged**